build/Signing.proj

```diff
@@ -38,6 +38,7 @@
       <!-- Built binaries -->
       <FilesToSign Include="$(OutputDirectory)/sdk/**/csc.exe;
                             $(OutputDirectory)/sdk/**/csc.dll;
+                            $(OutputDirectory)/sdk/**/VBCSCompiler.dll;
```

Comment:
This is just for auth signing correct, not strong name?

Reply:
Correct.

Strongname signing does not work on crossgenned assemblies. Any crossgen assembly will fail strong name signing check.

Reply:
> Any crossgen assembly will fail strong name signing check.

Is it a bug or by-design behavior?

Reply:
I think it's a bug in that tools that are used for strong-name signing (sn.exe) don't deal well with crossgen'ed images. There are related issues such as https://github.com/dotnet/sdk/issues/1502#issuecomment-324715367

It is possible to fix this, but it will be hard to roll out to everywhere that deals with strong-naming and the runtime that we're crossgening to target exclusively doesn't check strong name signatures so it's likely not worth addressing. cc @jkotas to correct me if I'm misunderstanding.